src/ui/public/selectable_row/selectable_row.js

```diff
@@ -0,0 +1,22 @@
+import { uiModules } from 'ui/modules';
+import template from './selectable_row.html';
+
+const app = uiModules.get('kibana');
+
+app.directive('selectableRow', function () {
+  return {
+    restrict: 'E',
+    replace: true,
+    transclude: true,
+    template: template,
+    scope: {
+      id: '=',
+      isSelected: '=',
+      onSelectChange: '=',
+    },
+    controllerAs: 'selectableRow',
```

Comment:
I kind of like the way that `this.isSelected` mirrors the way it's accessed within the context of the controller, but `this` in templates normally references `$scope`, so we would have to use `controllerAs: 'this'` and that might cause more confusion (especially for people used to accessing `$scope` with `this`) than it's worth... 

That said, naming is hard. My preference, in order, is:
1. `controllerAs: 'selectableRow'`
1. `controllerAs: 'controller'`
1. `controllerAs: 'this'`